src/swap/exolix.ts

```diff
@@ -151,41 +164,42 @@ export function makeExolixPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
             request.toCurrencyCode
           )
 
+    const quoteAmount = parseFloat(nativeQuoteAmount)
+
     const {
       fromCurrencyCode,
       toCurrencyCode,
       fromMainnetCode,
       toMainnetCode
     } = getCodes(request)
 
-    // The Exolix documentation doesn't detail this mainnetCode:currencyCode functionality
-    // but it's been verified by testing
-    const safeFromCurrencyCode = `${fromMainnetCode}:${fromCurrencyCode}`
-    const safeToCurrencyCode = `${toMainnetCode}:${toCurrencyCode}`
-
     // Swap the currencies if we need a reverse quote:
     const quoteParams =
       request.quoteFor === 'from'
         ? {
-            coin_from: safeFromCurrencyCode,
-            coin_to: safeToCurrencyCode,
-            deposit_amount: quoteAmount,
-            rate_type: 'fixed'
+            coinFrom: fromCurrencyCode,
+            coinFromNetwork: fromMainnetCode,
+            coinTo: toCurrencyCode,
+            coinToNetwork: toMainnetCode,
+            amount: quoteAmount,
+            rateType: 'fixed'
```

Comment:
Both of these `quoteParams` are identical. You need to differentiate FROM vs TO quotes.

Reply:
What do you mean? 
On line 156 we received `exchangeQuoteAmount` from `request.quoteFor = from|to` And if `request.quoteFor === 'to'` we will receive quoteAmount differen with quoteAmount as `request.quoteFor === 'from'`

Reply:
@paullinator Do we need any more updates from our side?